icefoganalytics/elcc-data-management
# ELCC-10: Add Database Modeling Diagrams

Depends on: 
- https://github.com/icefoganalytics/elcc-data-management/pull/49/files

Fixes ELCC-10

# Context

Its often much easier to see how tables in a database connect, and whether the database layout is sensible, when looking at a diagram.

Add some PlantUML diagrams that show the database tables and how they are linked.

# Implementation

Add plantuml server to docker compose development file.
Snake case some missed column namings.

# Testing Instructions

1. Boot the app via `dev up`

2. Check that the app complies, and that you can log in at http://localhost:8080.

3. Check that you can run the migration up and down via
```bash
dev migrate up
dev migrate down
dev migrate up
```
